pkg/model/api/config/authorization.go

```diff
@@ -18,17 +18,29 @@ import (
 
 	"github.com/perses/perses/pkg/model/api/v1/common"
 	"github.com/perses/perses/pkg/model/api/v1/role"
+	"github.com/sirupsen/logrus"
 )
 
 var (
 	defaultCacheInterval = 30 * time.Second
 )
 
+type KubernetesProvider struct {
+	Enabled    bool   `json:"enabled,omitempty" yaml:"enabled,omitempty"`
+	Kubeconfig string `json:"kubeconfig,omitempty" yaml:"kubeconfig,omitempty"`
+}
+
+type AuthorizationProviders struct {
+	EnableNative bool               `json:"enable_native" yaml:"enable_native"`
```

Comment:
as you mention in the PR's description, we can avoid the breaking change by simply saying when the boolean `security.enable_auth` is set to true while no provider is set, then the `native` one is enable.

Reply:
this should be done at the security config level